### PR TITLE
Fix: Back to search results button functionality

### PR DESCRIPTION
**Pull request by sam-sny**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Updated the 'Back to search results' button text and functionality. Replaced the original text with 'Back to search result' and implemented a function to close the panel when the button is clicked. This enhancement ensures a more intuitive user experience by providing clear navigation back to search results.

## Screenshots
Before: 
Screenshot (195)

After: 

patient-scheduled-visit:
Screenshot (191)

visit-form:
Screenshot (193)

## Related Issue
https://openmrs.atlassian.net/browse/O3-2467

### CODE REVIEW

**Review by mseaton — CHANGES_REQUESTED** [review]
@sam-sny seem my comments.  Also, can you please create a branch for your changes rather than issuing changes directly from main?

**Review by mseaton — CHANGES_REQUESTED** [review]
Not sure what you are presenting here @sam-sny , but I don't think it's intended for this issue.

File: package.json [hunk 1]
```diff
@@ -21,14 +21,15 @@
     "@hookform/resolvers": "^3.3.1",
     "classnames": "^2.3.2",
     "react-hook-form": "^7.46.2",
+    "react-joyride": "^2.8.0",
```
Comment (Vijaykv5) [inline package.json:24]: Why are you adding a third party library for fixing this issue? @sam-sny

**Comment by denniskigen** [comment]
Closing as stale

**Pull request closed without merging**